js-plugins/sass/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@farmfe/js-plugin-sass",
+  "version": "0.0.1",
+  "description": "support scss compile for farm.",
+  "main": "./dist/index.js",
```

Comment:
define `exports`, `repository`, `bugs`, `author` field in package.json, and set `type="module"`.